### PR TITLE
Fix #20338: Scenery Picker Hotkey not selected with re-paint tool.

### LINKED ISSUE #20338: Hotkey for scenery picker not working when re-paint tool is selected
### Operating System

Windows 10, 64-bit

### OpenRCT2 build

OpenRCT2, v0.4.5-66-g90b65cd (90b65cd on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This is a development issue

### Describe the issue

When the re-paint tool for scenery is selected, the shortcut key for the scenery picker doesn't react and re-paint tool stays selected.

### Steps to reproduce

1. select re-paint tool in the scenery window
2. press shortcut key for scenery picker

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
Followup for https://github.com/OpenRCT2/OpenRCT2/pull/21644.

This PR enables:

- Allows for the scenery picker hotkey with the re-paint tool.
- Always shows the scenery buttons.

Sorry that I had to close the last PR - when I was testing my changes I noticed that the scenery scatter window was not working and wanted to test if that was the same case on develop. So I stupidly merged develop into my branch and when I tried to undo my changes it deleted the branch so I cannot commit to it anymore.

~Note that the scenery scatter window appears to be broken on develop and is not related to this commit.~ Fixed in (https://github.com/OpenRCT2/OpenRCT2/pull/21649)

### CODE REVIEW

File: src/openrct2-ui/windows/Scenery.cpp [hunk 1]
```diff
@@ -630,10 +630,7 @@ static Widget WindowSceneryBaseWidgets[] = {
             if (gWindowSceneryScatterEnabled)
                 pressed_widgets |= (1uLL << WIDX_SCENERY_BUILD_CLUSTER_BUTTON);
 
-            widgets[WIDX_SCENERY_ROTATE_OBJECTS_BUTTON].type = WindowWidgetType::Empty;
-            widgets[WIDX_SCENERY_EYEDROPPER_BUTTON].type = WindowWidgetType::Empty;
             widgets[WIDX_SCENERY_BUILD_CLUSTER_BUTTON].type = WindowWidgetType::Empty;
-            widgets[WIDX_RESTRICT_SCENERY].type = WindowWidgetType::Empty;
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/Scenery.cpp:636]: This is not correct. _Only_ the line that removes the `WIDX_SCENERY_EYEDROPPER_BUTTON` should be dropped here.

File: src/openrct2-ui/windows/Scenery.cpp [hunk 2]
```diff
@@ -630,10 +630,7 @@ static Widget WindowSceneryBaseWidgets[] = {
...
 
             if (!(gWindowSceneryPaintEnabled & 1))
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/Scenery.cpp:635]: Remove this block.

**Review by Gymnasiast — COMMENTED** [review]
Tested, seems to work correctly. Needs rebasing and a changelog entry:
```
Fix: [#20338] Cannot select Scenery Picker or Scatter Tool when the scenery recolouring tool is active.
```

**Review by Gymnasiast — APPROVED** [review]

**Comment by Harry-Hopkinson** [comment]
Sorry about that was trying to rebase rather than merge and just deleted the branch, luckily managed to recover it.

**Review by Gymnasiast — APPROVED** [review]

**Review by tupaschoal — APPROVED** [review]

**Comment by Mmmmulder** [comment]
thank you for fixing this, much appreciated!

**Pull request merged**